netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -631,23 +661,31 @@ NettyServerBuilder setTransportTracerFactory(
 
   @Override
   public NettyServerBuilder useTransportSecurity(File certChain, File privateKey) {
```

Comment:
The default is plaintext for this server builder. Should we have an explicit `useInsecure` or `usePlaintext` similar to other use cases? I understand there is a backward compatibility issue (and not really related to this PR).

Reply:
I don't see any need to enhance the existing API. ServerCreds should replace it long-term.